config/routes.rb

```diff
@@ -1,3 +1,8 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  devise_for :members, controllers: { registrations: 'registrations' }
+  resources :posts
```

Comment:
## Optional Changes ♻️ 
- [ ] We might want to restrict these routes to the following 👇 for more compactness ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)).

```suggestion
  resources :posts, only: [:new, :create, :index\]
```